app/templates/components/comment-item.hbs

```diff
@@ -27,7 +27,10 @@
   </div>
 {{else}}
   <div class="comment-info">
-    {{#link-to 'slugged-route' comment.user.username class="username"}}{{comment.user.username}}{{/link-to}} commented {{moment-from-now comment.insertedAt}}
+    {{user/display-username
+      githubUsername=comment.user.githubUsername
+      username=comment.user.username
+    }} commented {{moment-from-now comment.insertedAt}}
```

Comment:
This might in the future end up depending on other factors to, not just `username` and `githubUsername`.

Since the purpose of the component is "displaying the correct username content for the user, depending on the information the user provides", maybe we should pass in the entire user, not just these two fields.

Reply:
Makes sense, especially given the conversation that I'd eventually like some metadata on the user that helps the logic be more explicit about their current identity provider.